docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2020,6 +2020,20 @@ ha_engine_address = 127.0.0.1:6379
 
 <hr>
 
+## [plugin.plugin_id]
+
+This section allows to configure plugin-specific settings. `plugin_id` must be replaced with the plugin id present in `plugin.json`.
```

Comment:
```suggestion
This section explains how to configure plugin-specific settings. Replace thge `plugin_id` attribute with the plugin ID present in `plugin.json`.
```

Reply:
I have committed the second part ("must be replaced" -> "replace the"), but I am not sure about the first one.

I was referring to the `[plugin.plugin_id]` ini file section, not the paragraph.

Do you think it's okay to leave it as "This section allows to configure plugin-specific settings" or maybe "This section can be used to configure plugin-specific settings" ?

Thanks 😄

Reply:
Okay, I misunderstood the sentence. You might want to reword to "This section can be used..." or "This section of the of the ini file..."